cl/alerts/utils.py

```diff
@@ -14,9 +15,15 @@
 )
 from cl.lib.command_utils import logger
 from cl.lib.elasticsearch_utils import add_es_highlighting
+from cl.lib.types import CleanData
+from cl.search.constants import (
+    ALERTS_HL_TAG,
+    SEARCH_RECAP_CHILD_HL_FIELDS,
+    recap_document_filters,
+    recap_document_indexed_fields,
+)
```

Comment:
We're not using any of these elements. Let's remove them to clean up the imports.

Reply:
correct, we can remove them